data/lang/ui-core/en.json

```diff
@@ -1095,6 +1095,10 @@
       "description" : "",
       "message" : "Ship is too small for the current crew."
    },
+   "TOO_SMALL_TO_TRANSSHIP" : {
+      "description" : "",
+      "message" : "Ship is too small to transship your cargo."
+   },
```

Comment:
What happens if you can't fit all the cargo?
:-)

On Wed, Jun 29, 2016 at 9:30 AM Karl F <notifications@github.com> wrote:

> In data/lang/ui-core/en.json
> <https://github.com/pioneerspacesim/pioneer/pull/3725#discussion_r68897427>
> :
>
> > @@ -1095,6 +1095,10 @@
> >        "description" : "",
> >        "message" : "Ship is too small for the current crew."
> >     },
> > +   "TOO_SMALL_TO_TRANSSHIP" : {
> > +      "description" : "",
> > +      "message" : "Ship is too small to transship your cargo."
> > +   },
>
> Also, never mind this. I looked it up, and learned something new.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pioneerspacesim/pioneer/pull/3725/files/272c0abc3aa81950b0449653d7e8a2e8e7358be4#r68897427>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAkONT-WDMYYWtoEwsOJ0ynNZhQhpkcnks5qQh8JgaJpZM4I94Hz>
> .
>